public/app/plugins/panel/singlestat/module.ts

```diff
@@ -215,6 +215,13 @@ class SingleStatCtrl extends MetricsPanelCtrl {
     this.render();
   }
 
+  onColorChange(panelColorIndex) {
+    return (color) => {
```

Comment:
React doesn't have two-way data binding, so we should use callbacks for updating model from view. Maybe you have better ideas, or it's OK?

Reply:
yes, this fine!